packages/graphql/src/translate/create-set-relationship-properties.ts

```diff
@@ -81,6 +82,19 @@ function createSetRelationshipProperties({
 
             return;
         }
+        
+        const mathMatch = matchMathField(key);
+        const {isMatched} = mathMatch;
+        if (isMatched) {
+            const mathDescriptor = mathDescriptorBuilder(value as number, relationship, mathMatch);
+            if (_properties.find(([_key]) => _key === mathDescriptor.dbName)) {
```

Comment:
Usually, leading underscore is reserved for unused variables. Maybe we should change this here:
```suggestion
            if (properties.find(([key]) => key === mathDescriptor.dbName)) {
```

Reply:
Good catch! I'll find a different naming, usually I use the underscore prefix to describe private properties, but in this case was a lazy attempt to avoid shadow variables with the outer scope  :)

Reply:
Same, but `private` keyword already exists in typescript, so no need for that